frontend/packages/console-app/src/components/quick-starts/QuickStartDrawer.tsx

```diff
@@ -27,14 +18,13 @@ type DispatchProps = {
 
 type QuickStartDrawerProps = StateProps & DispatchProps;
 
-const QuickStartDrawer: React.FC<QuickStartDrawerProps> = ({
-  children,
-  activeQuickStartID,
-  activeQuickStartStatus,
-  onClose,
-}) => {
+const QuickStartDrawer: React.FC<QuickStartDrawerProps> = ({ children }) => {
```

Comment:
looks like type definition `QuickStartDrawerProps` has extra props now here and even in other components which are no longer connected one